src/camera.py

```diff
@@ -0,0 +1,105 @@
+from os import getcwd
+
+from picar.SunFounder_PCA9685 import Servo
+from picar import filedb
+
+
+'''
+Camera class to control panning and tilting.
+
+Creates camera object that can be operated on using pan() and tilt().
+pan() operates [0,180] degrees. tilt() operates [0,150] degrees. These 
+functions can be calibrated using calibrate_pan() and calibrate_tilt().
+Calibration adds values to the pan() or tilt().
+'''
+class Camera:
+    PAN_CHANNEL = 1
+    TILT_CHANNEL = 2
+    MAX_PAN_DEGREE = 180
+    MAX_TILT_DEGREE = 150
+    MIN_DEGREE = 0
+    OFFSET_TO_SERVO = 0
+
+    def __init__(self, bus_number=1):
+        self._DB = filedb.fileDB(getcwd() + 
+                                 "/../SunFounder_PiCar-V/remote_control/remote_control/driver/config")
+        self._pan_offset = int(self._DB.get('pan_offset', default_value=0))
+        self._tilt_offset = int(self._DB.get('tilt_offset', default_value=0))
+        self._pan_servo = Servo.Servo(self.PAN_CHANNEL,  
+                                      bus_number=bus_number, 
+                                      offset=self.OFFSET_TO_SERVO)
```

Comment:
Why is this `OFFSET_TO_SERVO` instead of the value that you just grabbed?

Reply:
Do we also want that behavior though? I don't know if we should be messing with the PWM stuff if they need it.